nexus/src/external_api/views.rs

```diff
@@ -36,6 +36,88 @@ impl Into<Silo> for model::Silo {
     }
 }
 
+// IDENTITY PROVIDER
+
+#[derive(Clone, Copy, Debug, Deserialize, Serialize, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum IdentityProviderType {
+    /// SAML identity provider
+    Saml,
+}
+
+impl Into<IdentityProviderType> for model::IdentityProviderType {
+    fn into(self) -> IdentityProviderType {
+        match self {
+            model::IdentityProviderType::Saml => IdentityProviderType::Saml,
+        }
+    }
+}
+
+/// Client view of an ['IdentityProvider']
```

Comment:
Don't know if it matters, but all the other `[]` things in doc comments have backticks:

```suggestion
/// Client view of an [`IdentityProvider`]
```

Reply:
ah, fixed in 9ac15729